tests/functional/test_mandel.py

```diff
@@ -150,12 +150,12 @@ def test_scaled_vs_unscaled_systems():
         "solid": pp.SolidConstants(mandel_solid_constants),
     }
     time_manager_unscaled = pp.TimeManager([0, 10], 10, True)
-    params_unscaled = {
+    model_params_unscaled = {
         "material_constants": material_constants_unscaled,
         "time_manager": time_manager_unscaled,
     }
-    unscaled = MandelSetup(params=params_unscaled)
-    pp.run_time_dependent_model(model=unscaled, params=params_unscaled)
+    unscaled_model = MandelSetup(params=model_params_unscaled)
```

Comment:
I would suggest the same logic in the naming for both the params dictionary as well as the model. So if using `model_params_unscaled` I would also go for `model_unscaled`. Maybe this is just my taste?